Assets/Shopify/UIToolkit/Components/RemoteImageLoader.cs

```diff
@@ -10,10 +10,18 @@ namespace Shopify.UIToolkit {
     /// </summary>
     [RequireComponent(typeof(Image))]
     public class RemoteImageLoader : MonoBehaviour {
+
+        /// <summary>
+        /// When true, downloaded images will be cached.
+        /// </summary>
+        public bool UseCache;
```

Comment:
Is there a situation where caching is not an overall win? at the very least this should default to true imo.

Reply:
Setting the default to true makes sense 👍